attorch/module.py

```diff
@@ -23,9 +23,15 @@ def __len__(self):
         return len(self._modules)
 
     def __iter__(self):
-        return iter(self._modules.values())
+        return iter(self._modules.keys())
 
     def keys(self):
         yield from self._modules.keys()
```

Comment:
This should be changed to `return` as well. `self._modules.keys()` already returns an iterable so you can just return it.